tests/unit/s2n_x509_validator_test.c

```diff
@@ -1141,9 +1132,8 @@ int main(int argc, char **argv) {
 
         uint8_t *raw_data = (uint8_t *)s2n_stuffer_raw_read(&ocsp_data_stuffer, ocsp_data_len);
 
-        EXPECT_EQUAL(S2N_CERT_ERR_INVALID, s2n_x509_validator_validate_cert_stapled_ocsp_response(&validator, connection,
-                                                                                                  raw_data,
-                                                                                                  ocsp_data_len));
+        EXPECT_ERROR_WITH_ERRNO(s2n_x509_validator_validate_cert_stapled_ocsp_response(&validator, connection,
+                raw_data, ocsp_data_len), S2N_ERR_CERT_UNTRUSTED);
```

Comment:
Is "S2N_ERR_CERT_UNTRUSTED" more accurate here than "S2N_CERT_ERR_INVALID" was?

Reply:
I believe it is - this line fails because the certificate's issuer can't be found. So, it seems like the certificate is still valid in this case, but can't be trusted because there's no issuer. Or does this make the certificate invalid?